stylesheets/styles.css

```diff
@@ -69,13 +69,17 @@ body, html {
 
 /* @media(min-width: 360px) {
 
-}
+} */
 
 @media(min-width: 768px) {
-
+    .logo {
+        display: flex;
+        height: 140%;
+        width: 65rem;
+    }
```

Comment:
Could we possibly change the height to 100%? At 140%, it overflows from the 'header' container. Otherwise, this looks fine! 🙂

Reply:
We will also want to set the height to be no more than 100% for the desktop version as well.  The % refers to the height of the containing element, so setting the height to be more than 100% will always cause it overflow from the container.

Reply:
Got it. I also used "object-fit" property to assure the image remains proportional to the original.

Reply:
I agree with @clinturbin's assessment. We will want to make sure that the desktop breakpoint is set at height: 100%. 

```
@media(min-width: 1200px) {
.logo {
    display: flex;
    height: 100%;
}
```